conf/layer.conf

```diff
@@ -19,7 +19,7 @@ BBFILE_PATTERN_meta-python-ai := "^${LAYERDIR}/"
 BBFILE_PRIORITY_meta-python-ai = "8"
 
 LAYERVERSION_meta-python-ai = "1"
-LAYERSERIES_COMPAT_meta-python-ai = "styhead walnascar"
+LAYERSERIES_COMPAT_meta-python-ai = "whinlatter"
```

Comment:
Apparently there was a recent-ish breaking change in whinlatter that makes the build system complain if `S` is set. I would have liked to keep compatibility with walnascar, but oh well.